internal/controllers/extension_controller.go

```diff
@@ -55,10 +56,11 @@ import (
 type ExtensionReconciler struct {
 	client.Client
 	BundleProvider BundleProvider
-	HasKappApis    bool
 }
 
-var errkappAPIUnavailable = errors.New("kapp-controller apis unavailable on cluster")
+var (
+	bundleVersionKey = "olm.bundle-version"
```

Comment:
I feel like I might be missing a best-practice for this sort of thing; let me know if there's a better way!

Reply:
I would suggest `olm.operatorframework.io/bundleVersion`